### PR TITLE
feat: login page

### PR DESCRIPTION
**Pull request by xappyyy**

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1,3 +1,23 @@
 export default function Home() {
-  return <div>this is first page</div>;
+  return (
+    <div className="flex h-screen w-screen flex-col">
+      <div className="flex flex-grow flex-col items-center justify-center">
+        <div className="bg-red-700 border-red-200 relative flex h-1/3 w-4/6 max-w-lg flex-col items-center justify-center rounded-3xl border-4 border-solid">
+          <div className="right-10px absolute -top-14 h-28 w-28 rounded-full bg-black sm:-top-16 sm:h-32 sm:w-32"></div>
+          <div className="flex h-1/3 w-5/6 min-w-fit flex-col">
+            <div className="text-xl text-white sm:text-2xl">เลขรหัสนิสิต</div>
+            <div className="h-1/3 w-full rounded-xl bg-gray-400"></div>
```
Comment (shalluv) [inline src/pages/index.tsx]: I assume that this is an input, right? Please use the `<input />` tag.

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,3 +1,23 @@
 export default function Home() {
-  return <div>this is first page</div>;
+  return (
+    <div className="flex h-screen w-screen flex-col">
```
Comment (shalluv) [inline src/pages/index.tsx]: This is okay, but I prefer using `min-h-screen` and `w-full` instead of these.

File: src/pages/index.tsx [hunk 3]
```diff
@@ -1,3 +1,23 @@
...
+            <div className="h-1/3 w-full rounded-xl bg-gray-400"></div>
+          </div>
+          <div className="absolute bottom-5 h-1/6 w-1/5 min-w-fit bg-white sm:w-1/6">
+            <div className="text-center text-xl text-black sm:text-2xl">
+              login
+            </div>
+          </div>
+        </div>
+      </div>
+      <div className="bg-red-500 flex h-32 items-center justify-center">
```
Comment (shalluv) [inline src/pages/index.tsx]: There is no need to import Footer here, as Tar has already included it in the `_app.tsx` files.

Comment (shalluv) [reply src/pages/index.tsx]: Just add `pb-32` to the top-most div and it will be fine.

File: src/pages/index.tsx [hunk 4]
```diff
@@ -1,3 +1,23 @@
 export default function Home() {
-  return <div>this is first page</div>;
+  return (
+    <div className="flex h-screen w-screen flex-col">
+      <div className="flex flex-grow flex-col items-center justify-center">
+        <div className="bg-red-700 border-red-200 relative flex h-1/3 w-4/6 max-w-lg flex-col items-center justify-center rounded-3xl border-4 border-solid">
+          <div className="right-10px absolute -top-14 h-28 w-28 rounded-full bg-black sm:-top-16 sm:h-32 sm:w-32"></div>
```
Comment (shalluv) [inline src/pages/index.tsx]: The class name "right-10px" has no effect. If you intended to use an arbitrary value, you may want to use "right-[10px]" instead. 

However, I suggest using Tailwind classes that are as close to 10 as possible, but not necessarily exactly 10, for better results.

Anyway, there is no need to adjust this `div` to the right.

File: src/pages/index.tsx [hunk 5]
```diff
@@ -1,3 +1,23 @@
 export default function Home() {
-  return <div>this is first page</div>;
+  return (
+    <div className="flex h-screen w-screen flex-col">
+      <div className="flex flex-grow flex-col items-center justify-center">
+        <div className="bg-red-700 border-red-200 relative flex h-1/3 w-4/6 max-w-lg flex-col items-center justify-center rounded-3xl border-4 border-solid">
```
Comment (shalluv) [inline src/pages/index.tsx]: I just added red to the Tailwind configuration, so now you should use `bg-red` and `border-red/20` to accurately match the design.

Anyway, this is not your fault. As I said, I just added them.

**Review by shalluv — CHANGES_REQUESTED** [review]
Furthermore, there are many unnecessary nested divs. Please double-check where simplification is possible.

At this point, you should insert the logo to replace the gray circle div.

**Comment by xappyyy** [comment]
Please help me on background and logo. Also, border-red/20 doesn't work somehow. I'm dying laew.

**Pull request merged**